src/main/webapp/displaycharities.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>User Page</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="/js/display-charities.js"></script>
+</head>
+<body >
+<nav>
+    <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/aboutus.html">About Our Team</a></li>
```

Comment:
The "About Our Team" is not on the nav bar anymore; To see the current nav bar you can use other recent files like public feed